formatters/formatters.go

```diff
@@ -0,0 +1,15 @@
+package formatters
```

Comment:
Hm.  I am not sure about putting them in a separate package.  Is that necessary?

Reply:
```
import (
  "some/platformsh"
  "some/platformsh/library/mysql"
)

func main() {
   mysql.Connect("database")
}
```
based on this comment from @demosdemon I thought that's what he meant by making it a subpackage.

Reply:
This way rather than having a `mysql.go` and a `postgresql.go` a user imports `...config-reader-go/formatters` and then uses whatever function they need. Better plan?